gw_eccentricity/gw_eccentricity.py

```diff
@@ -126,23 +130,29 @@ def measure_eccentricity(tref_in=None,
           quasicircular counterpart from the omega22(t) of the eccentric
           waveform.
         - "AmplitudeFits": Uses Amp22(t) and iteratively subtracts a
-          PN-inspired fitting function from it, and finds extrema of the
-          residual.
+          PN-inspired fit of the extrema of Amp22(t) from it, and finds extrema
+          of the residual.
         - "FrequencyFits": Uses omega22(t) and iteratively subtracts a
-          PN-inspired fitting function from it, and finds extrema of the
-          residual.
+          PN-inspired fit of the extrema of omega22(t) from it, and finds
+          extrema of the residual.
         Default is "Amplitude".
-        Available list of methods can be also obtained from
+        The available list of methods can be also obtained from
         gw_eccentricity.get_available_methods().
+        Detail description of these methods can be found in our paper,
+        see Sec. III of arxiv.xxxx.xxxx.
 
         The Amplitude and Frequency methods can struggle for very small
-        eccentricities (~1e-3), especially near the merger, as the secular
-        amplitude/frequency growth dominates the modulations due to
-        eccentricity, making extrema finding difficult. This is the main reason
-        for using the residual methods,
-        ResidualAmplitude/ResidualFrequency/FrequencyFits, which first remove
-        the secular growth before finding extrema. However, methods that use
-        the frequency for finding extrema
+        eccentricities (~1e-2...1e-3), especially near the merger, as the
+        secular amplitude/frequency growth dominates the modulations due to
+        eccentricity, making extrema finding difficult. See Eq.(25) of
+        arxiv.xxxx.xxxx to get an estimate of the smallest eccentricity that
+        can be measured using the Amplitude/Frequency method.
```

Comment:
Nah, remove the estimate thing here. We want to keep the docs as short as possible, and this estimate is too much of an order-of-mag-estimate that this will be more of a distraction. Real fans will always find it in the paper.

Reply:
Done